lib/matplotlib/tests/test_colors.py

```diff
@@ -70,6 +70,73 @@ def test_register_cmap():
         cm.register_cmap()
 
 
+@pytest.mark.skipif(matplotlib.__version__ < "3.5",
```

Comment:
I'm hesitant adding tests for future intended behavior. We don't know for sure if this will be implemented for the version and with the behavior we anticipate now.

Additionally, the test will be ignored during the complete development phase (master version targeting 3.3 is currently evaluated to `3.2.1.post2103+gb94812c60`). The check would only hit very late during the release.

You may keep this, but it needs explanation. If we are hit by the failing test in a year or so, we need a clear way of finding out why this is here, why the tested functionality is desired and what we have to do to implement it. Therefore, please add a comment describing the intention and referring to the modification roadmap discussed on github.

Reply:
Yeah, that is unfortunate that this wouldn't actually hit until near the release. Is there a way to test against something else? My thought is that leaving these there will make sure this doesn't get lost along the way and it isn't hard to implement these since I've already got the code in another linked PR. I put a link to the gh issue number in as a comment now.

Reply:
Other tests could be earlier but would still hit at some other time and break CI, when we maybe don't want to implement this right now.

Either leave it like this or remove it here and put the test just into your PR.

Reply:
It is better to put the test in the PR where the change is actually made.

There are better ways to document a future test than a skipped test (I don't think this would actually get run until we tagged the rc.  

As a side note, In general comparing versions via text is not the best way as it sorts pre-releases as being later than the final releases, `distuitls.version.LooseVersion` is usually the better option.

Reply:
Thanks for that suggestion of comparison with distutils. I just removed all of these future tests. I'll hope it doesn't drop off my radar.